scripts/meta/setup.mjs

```diff
@@ -45,3 +45,14 @@ runInContext(
 			encoding: "utf-8"
 		}).status
 );
+
+runInContext(
+	"corepack prepare pnpm@8.2.0",
```

Comment:
I thought `corepack` would install a proper version after `corepack enable`, do we need `corepack prepare` explicitly?

Reply:
> I thought `corepack` would install a proper version after `corepack enable`, do we need `corepack prepare` explicitly?

you are correct, it is not needed and now reverted